### PR TITLE
Phone Number filter malfunction resolved and minor UI fixes

### LINKED ISSUE #4267: Phone number filter malfunction and UI fixes
**Describe the bug**
When I click on the cross of the `Phone number` filter field on the Patients page, the filter is removed but the badge is still visible with the value `+91`
https://www.loom.com/share/bb2d51999a3145cbb218040bb5b95668
image

- Buttons should be full width in mobile view on the Patients page
https://care.coronasafe.in/patients
image

- `No Notices` text should be full width on the Notice Board page
https://care.coronasafe.in/notice_board
image

**To Reproduce**
Go to the Patients page. Add some value in the Phone Number filter field. Then click the cross next to it.

**Expected behavior**
The filter badge should also disappear when cross is clicked

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

### PR DESCRIPTION
**Pull request by Pranshu1902**
Fixes #4267 

- Pressing the cross removes the badge as well on the Patients page
https://www.loom.com/share/3731b073a82f417b816694437030cb06
- Made buttons full-width in mobile view on the Patient's page
http://localhost:4000/patients
image

- Made `No Notices` text full width on the Notice Board page
http://localhost:4000/notice_board
image

### CODE REVIEW

**Comment by nihal467** [comment]
LGTM

**Comment by khavinshankar** [comment]
@Pranshu1902 can you merge this with the develop branch and after merging if cypress fails, please do resolve that

**Review by khavinshankar — APPROVED** [review]

**Comment by Pranshu1902** [comment]
@khavinshankar done!

**Pull request merged**
